docs/timezone.md

```diff
@@ -0,0 +1,426 @@
+- "UTC"
```

Comment:
Can we move this file into the `pluginreceiver` directory? I'd like to keep high level docs in the `docs` directory. Also, can we add a title and brief summary?

Something like:

```markdown
# Timezone

Below is a list of possible values for the `timezone` plugin parameter:
```

Reply:
My mistake, just saw all the md files in docs and assumed it was supposed to be there